src/main/java/org/mastodon/mamut/clustering/ClusterRootNodesController.java

```diff
@@ -284,15 +284,15 @@ public List< String > getFeedback()
 		List< String > feedback = new ArrayList<>();
 		if ( cropStart >= cropEnd )
 		{
-			String message = "Crop start (" + cropStart + ") must be smaller than crop end (" + cropEnd + ")";
+			String message = "Crop start (" + cropStart + ") must be smaller than crop end (" + cropEnd + ").";
 			feedback.add( message );
 			logger.debug( message );
 		}
 
 		int roots = getRoots().size();
-		if ( numberOfClasses >= roots )
+		if ( numberOfClasses > roots )
 		{
-			String message = "Number of classes (" + numberOfClasses + ") must be smaller than number of valid roots (" + roots + ")";
+			String message = "Number of valid roots (" + roots + ") must be smaller than the number of classes (" + numberOfClasses + ").";
```

Comment:
The text is wrong now. Die you mean "Number of valid roots must be greater or equal to the number of classes."?

Reply:
Yes indeed. This was wrong. I changed it to `Number of classes must not be greater than the number of valid roots`, which means the same.
Thanks for spotting.